symphony/bdk/core/service/message_service.py

```diff
@@ -0,0 +1,366 @@
+from symphony.bdk.core.auth.auth_session import AuthSession
+from symphony.bdk.gen.agent_api.attachments_api import AttachmentsApi
+from symphony.bdk.gen.agent_api.messages_api import MessagesApi
+from symphony.bdk.gen.agent_model.v4_import_response import V4ImportResponse
+from symphony.bdk.gen.agent_model.v4_imported_message import V4ImportedMessage
+from symphony.bdk.gen.agent_model.v4_message import V4Message
+from symphony.bdk.gen.agent_model.v4_message_blast_response import V4MessageBlastResponse
+from symphony.bdk.gen.agent_model.v4_message_import_list import V4MessageImportList
+from symphony.bdk.gen.api_client import Endpoint
+from symphony.bdk.gen.model_utils import (
+    file_type
+)
+from symphony.bdk.gen.pod_api.default_api import DefaultApi
+from symphony.bdk.gen.pod_api.message_api import MessageApi
+from symphony.bdk.gen.pod_api.message_suppression_api import MessageSuppressionApi
+from symphony.bdk.gen.pod_api.pod_api import PodApi
+from symphony.bdk.gen.pod_api.streams_api import StreamsApi
+from symphony.bdk.gen.pod_model.message_metadata_response import MessageMetadataResponse
+from symphony.bdk.gen.pod_model.message_receipt_detail_response import MessageReceiptDetailResponse
+from symphony.bdk.gen.pod_model.message_status import MessageStatus
+from symphony.bdk.gen.pod_model.message_suppression_response import MessageSuppressionResponse
+from symphony.bdk.gen.pod_model.stream_attachment_item import StreamAttachmentItem
+
+
+class MultiAttachmentsMessagesApi(MessagesApi):
```

Comment:
is it ok to put both classes in the same file? to keep it simple would not it be simpler to have a separate file?

Reply:
The file containing the MessagesApi is generated from the specs and we shouldn't modify it, that's why I keep this class here.

Reply:
and we cannot move it to its own file?